scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -391,3 +433,59 @@ VQFN-28-1EP_4x5mm_P0.5mm_EP2.55x3.55mm:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+
+VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
+  device_type: 'VQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/32L_VQFN_5x5x0_9mm_3_1EP_RXB_C04-21395a.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 4.9
+    maximum: 5.1
+  body_size_y:
+    minimum: 4.9
+    maximum: 5.1
+  body_height:
+    maximum: 1
```

Comment:
Given as min/nom/max in the datasheet.

maximum is 0.9 by the way.

Reply:
The datasheet link was wrong, new values according to the correct datasheet.